blockx/chain.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "blockx-atlantis-testnet",
+  "chain_id": "blockx_190-1",
+  "pretty_name": "BlockX-Atlantis-Testnet",
+  "website": "https://www.blockxnet.com/",
+  "status": "live",
+  "network_type": "testnet",
+  "bech32_prefix": "blockx",
```

Comment:
I recommend registering this prefix before another chain takes it. https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Kindly elaborate more on this, excuse my newbie lack of knowledge.
You may just point out some tip/hint to learn more and understand what's required here & why?
Thanks in advance!

Reply:
For mainnets, it is required to register bech32 prefixes here in the other repo.

Reply:
Thank you very much!


Registered & awaiting merge https://github.com/satoshilabs/slips/pull/1738.